src/base/config.scss

```diff
@@ -136,8 +136,9 @@ $tu-large-lineHeight:         scaleFont($lineHeight, 2) !default;
 
 // * 3. Spacing
 
-$su-xxsmall:   scaleGrid(.25) !default; // 2px
-$su-xsmall:     scaleGrid(.5) !default; // 4px
+$su-xxxsmall:   scaleGrid(-3) !default; // 1px
+$su-xxsmall:    scaleGrid(-2) !default; // 2px
+$su-xsmall:     scaleGrid(-1) !default; // 4px
```

Comment:
I like this change... I wasn't following the whole previous `scaleGrid` thread, but this treatment feels more intuitive to me.